AppConnectSwift/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let dir = NSFileManager.defaultManager().URLsForDirectory(NSSearchPathDirectory.DocumentDirectory, inDomains: NSSearchPathDomainMask.UserDomainMask).last
         let key = "12345678901234567890123456789012".dataUsingEncoding(NSUTF8StringEncoding)
         // TODO: Substitute the apiToken value with proper token
```

Comment:
We'll actually want to remove the token here, and invalidate it. This shouldn't have been committed in the first place.